etc/profile-m-z/telegram.profile

```diff
@@ -16,9 +15,7 @@ include disable-programs.inc
 include disable-shell.inc
 include disable-xdg.inc
 
-mkdir ${HOME}/.TelegramDesktop
```

Comment:
There might still telegram versions in use which do use this directory (https://packages.debian.org/buster/telegram-desktop).

Reply:
Oh, I tried to figure out in what conditions Telegram would use that directory, but found nothing relevant. Makes sense.